azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/UpperMessageBarNotificationModel.kt

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models
+
+import android.view.View
+
+internal class UpperMessageBarNotificationModel {
+    var notificationIconId: Int = 0
+    var notificationMessageId: Int = 0
+    var notificationView: View? = null
```

Comment:
would it be possible to not use view in models? This will help to avoid memory leaks